### PR TITLE
Unix => Linux

### PR DESCRIPTION
**Pull request by troydai**
@victorhurdugaci @davidfowl

### CODE REVIEW

**Comment by victorhurdugaci** [comment]
:shipit:

**Comment by troydai** [comment]
The tests failed on _AppVeyor_ because of issue https://github.com/aspnet/Testing/issues/105. 

/cc @dougbu @rynowak

File: test/Microsoft.Framework.TestHost.Tests/TestHostTest.cs [hunk 1]
```diff
@@ -26,7 +25,8 @@ public TestHostTest()
             _testProject = Path.GetDirectoryName(libraryManager.GetLibraryInformation("Sample.Tests").Path);
         }
 
-        [Fact]
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```
Comment (dougbu) [inline test/Microsoft.Framework.TestHost.Tests/TestHostTest.cs]: add a `SkipReason` setting for these tests, referring to #104 or #105

**Comment by dougbu** [comment]
:shipit: with smallish additions

**Comment by troydai** [comment]
Fixing https://github.com/aspnet/Testing/issues/105

File: test/Microsoft.Framework.TestHost.Tests/TestHostTest.cs (hunk 1 above)
Comment (troydai) [inline test/Microsoft.Framework.TestHost.Tests/TestHostTest.cs]: ConditionalFact doesn't support Reason. These tests are runnable on Windows.

Comment (dougbu) [inline test/Microsoft.Framework.TestHost.Tests/TestHostTest.cs]: `[OSSkipCondition]` supports `SkipReason`

**Comment by troydai** [comment]
Ladies and Gentlemen, I'm going to merge this to dev shortly.

**Pull request merged**